plans/feat-beat-local-refs.md

```
@@ -0,0 +1,288 @@
+
+| 生成対象 | reference に渡せるもの | 渡せないもの |
+|---------|----------------------|-------------|
+| 画像生成 (`imageGenerator`) | `imageRefs` の画像（複数可） | `movieRefs` の動画 |
+| 動画生成 (`movieGenerator`) | `imageRefs` の画像（1枚） | `movieRefs` の動画 |
+| html_tailwind / markdown | `imageRefs` (`image:name`) + `movieRefs` (`movie:name`) | — |
+
+**結論**: `imageRefs`（画像パス）は全ての場面で使える万能 ref。`movieRefs`（動画パス）は html/markdown 内の `movie:name` でのみ使える。
+
+## Current Flow (Phase 1 完了後)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)